src/GitHub.VisualStudio/Services/UsageService.cs

```diff
@@ -100,6 +143,10 @@ public IDisposable StartTimer(Func<Task> callback, TimeSpan dueTime, TimeSpan pe
                     Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData),
                     program.ApplicationName,
                     StoreFileName);
+                userStorePath = Path.Combine(
+                    Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData),
```

Comment:
And I realize a lot of this is driven here.
Running this test locally pulls up my current user's `ApplicationData`.
How would I go about abstracting the file...

Reply:
If this class used Rothko, it would be simple to abstract. Rothko provides some environment abstractions so at least you can return a different path for this even if you go through regular IO. That would be a very minimal change that would make this easier to test I think?